### PR TITLE
configured tag sub model

### LINKED ISSUE #93: [backend] Events should have tags
Tags will be a new submodel inside the organisation app

### PR DESCRIPTION
**Pull request by ZenabKhan**
Resolves #93

### CODE REVIEW

File: event/models.py [hunk 1]
```diff
@@ -23,6 +23,12 @@ class EventSponser(models.Model):
     def __str__(self):
         return self.name
 
+class EventTag(models.Model):
+    name = models.CharField(max_length=255)
```
Comment (mashhoodr) [inline event/models.py]: should we have name and label as separate? @abdulmoizeng

File: event/serializers.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 from rest_framework import serializers
```
Comment (mashhoodr) [inline event/serializers.py]: @ZenabKhan need to also change the event seriliazer?

Comment (ZenabKhan) [reply event/serializers.py]: @mashhoodr you mean to add EventTag in serializer?

File: event/models.py (hunk 1 above)
Comment (abdulmoizeng) [reply event/models.py]: i would suggest **label & description** will look more meaningful here

File: event/serializers.py (hunk 2 above)
Comment (mashhoodr) [reply event/serializers.py]: yes so it returns the tag object not just the id

**Comment by mashhoodr** [comment]
@ZenabKhan dont forget to document in postman collection

**Pull request merged**